lib/ChangeDueDateDialog/ChangeDueDate.js

```diff
@@ -173,9 +175,12 @@ class ChangeDueDate extends React.Component {
         const putEndpoint = '/circulation/loans/';
 
         if (error.url && error.url.indexOf(putEndpoint)) {
-          const failedLoan = error.url.split(putEndpoint)[1];
+          const failedLoanId = error.url.split(putEndpoint)[1];
+          const errorMessage = this.itemIsDeclaredLost(failedLoanId) ?
+            <FormattedMessage id="stripes-smart-components.cddd.itemDeclaredLost" /> :
+            <FormattedMessage id="stripes-smart-components.cddd.changeFailed" />;
```

Comment:
```
 const errorMessage = this.itemIsDeclaredLost(failedLoanId) 
    ? <FormattedMessage id="stripes-smart-components.cddd.itemDeclaredLost" />
    : <FormattedMessage id="stripes-smart-components.cddd.changeFailed" />;
```
Please, use more readable way of transposition in a multi-line ternary

Reply:
Thanks, I will do it that way.